conf/up4.bess

```diff
@@ -227,6 +228,32 @@ if parser.enable_slice_metering:
     sliceMeter.set_default_gate(gate=m_fail)
     _in = sliceMeter
 
+# TODO: decide whether UE policing module is optional
+if True:
+    # sliceMeter enforces a per UE, per direction meter rate limit
+    ueMeter::Qos(fields=[{'attr_name':'src_iface', 'num_bytes':1}, \
```

Comment:
My interpretation of the PFCP spec is that the control plane can arbitrarily aggregate flows (identified by PDRs) through one or more QERs in sequence. So the fact that we do metering at the UE level is a control plane choice.

This could be a valid configuration (but I would like @badhrinathpa to confirm):
<img width="795" alt="My Sketches 2 - 2021-09-22 16 16 19" src="https://user-images.githubusercontent.com/3025473/134434762-32845b84-77f3-40ef-b97c-f0e72deee5cd.png">

For this reason, and for consistency with the existing `qerLookup` module, I would rename this to `qerLookup2`
```suggestion
    qerLookup2::Qos(fields=[{'attr_name':'src_iface', 'num_bytes':1}, \
```

Reply:
We decided to do metering based on fseid and src_iface, ignoring QER IDs. It will be up to PFCP agent to figure out if and when to configure the per-UE meter based on the QERs it sees. Closing this.